### PR TITLE
Migrate images to s3

### PR DESCRIPTION
**Pull request by AbraaoMota**
This Pull request is a mix of automated image migrations from sources we can safely migrate from (imgur and tinypic) and a lot of s3ify'ing of images manually of other images.
Any others will be up to @GramMcCram to make a decision on once this PR is merged.

### CODE REVIEW

File: source/blog/2014-06-07-jquery-uk-conference-2014.html.markdown [hunk 1]
```diff
@@ -60,13 +60,13 @@ In summary, conferences are awesome because...
 
 2) You see people speak who's name you're used to reading on API docs
 
-![Yehuda Katz talking about Why you should use Ember](https://lh3.googleusercontent.com/-7nLEdbR86Ik/U3nNAKVHUVI/AAAAAAAACYg/QPUWY9kpnZM/w1848-h1106-no/2014-05-19)
+![Yehuda Katz talking about Why you should use Ember](https://s3-eu-west-1.amazonaws.com/unboxed-web-image-uploader/6adc3d385da5eec7ae7a59a1370f4744.)
```
Comment (tomsabin) [inline source/blog/2014-06-07-jquery-uk-conference-2014.html.markdown]: This image is missing a file extension, but the link works..

File: source/blog/2014-06-07-jquery-uk-conference-2014.html.markdown [hunk 2]
```diff
@@ -60,13 +60,13 @@ In summary, conferences are awesome because...
...
 
-![Yehuda Katz talking about Why you should use Ember](https://lh3.googleusercontent.com/-7nLEdbR86Ik/U3nNAKVHUVI/AAAAAAAACYg/QPUWY9kpnZM/w1848-h1106-no/2014-05-19)
+![Yehuda Katz talking about Why you should use Ember](https://s3-eu-west-1.amazonaws.com/unboxed-web-image-uploader/6adc3d385da5eec7ae7a59a1370f4744.)
 
 (Yehuda Katz on Ember.js)
 
 3) You meet like-minded people
 
-![Nerdvana: the Retro Gaming lounge](https://lh3.googleusercontent.com/-6IOK5mpmC8g/U3nOBIWfzdI/AAAAAAAACY0/Z5_W5WNX5xg/w1848-h1106-no/2014-05-19)
+![Nerdvana: the Retro Gaming lounge](https://s3-eu-west-1.amazonaws.com/unboxed-web-image-uploader/2f5971dd15c35a30c0393bcfacbdc783.)
```
Comment (tomsabin) [inline source/blog/2014-06-07-jquery-uk-conference-2014.html.markdown]: This image is missing a file extension, but the link works..

**Review by tomsabin — COMMENTED** [review]
This looks good to me. There's only the two images as commented which do not have file extensions, but the image link still works. I'm going to try and manually fix this.

<details>
<summary>
bash script used to check files (first by curl HEAD only request & then download)
</summary>

`
awk -F: '/https:\/\/s3-eu-west-1.amazonaws.com\/unboxed-web-image-uploader\/*/{print $0}' ~/Sites/unboxed.co/source/blog/* | grep -Eoi '(https:\/\/s3-eu-west-1.amazonaws.com.*\w{32}\.(jpeg|jpg|png|gif)?)' | while read url; do curl -O $url; done
`
</details>

File: source/blog/2014-06-07-jquery-uk-conference-2014.html.markdown (hunk 1 above)
Comment (tomsabin) [reply source/blog/2014-06-07-jquery-uk-conference-2014.html.markdown]: I've manually renamed this in the AWS console to include the file extension (.jpg)

File: source/blog/2014-06-07-jquery-uk-conference-2014.html.markdown (hunk 2 above)
Comment (tomsabin) [reply source/blog/2014-06-07-jquery-uk-conference-2014.html.markdown]: I've manually renamed this in the AWS console to include the file extension (.jpg)

**Comment by ubxd** [comment]
Deploy preview for *unboxed-co-production* ready!

Built with commit faff2deabe3767fe2d28e5bb169942a0b76e3751

https://deploy-preview-349--unboxed-co-production.netlify.com

**Pull request merged**